src/components/CartHeader.js

```diff
@@ -6,8 +6,11 @@ import { Link } from 'react-router-dom';
 export function CartHeader({ cart }) {
   const cartItems = cart ? cart.length : 0;
   return (
-    <Link to="/myCart">
-      <button id="cart-button">Cart:{cartItems}</button>
-    </Link>
+    <div className='cart-header'>
+      <Link to='/myCart'>
+        <img id='cart-icon' src='/img/cart.png'></img>
```

Comment:
Is `/myCart` a valid link?
I forget

Reply:
shoot - I didnt mean to merge

Reply:
Yup yup, /myCart is valid. Or at least working on my machine